evalml/tests/data_checks_tests/test_data_checks.py

```diff
@@ -229,24 +227,13 @@ def get_expected_messages(problem_type):
             message="1 row(s) (20.0%) of target values are null",
             data_check_name="InvalidTargetDataCheck",
             message_code=DataCheckMessageCode.TARGET_HAS_NULL,
-            details={"num_null_rows": 1, "pct_null_rows": 20.0},
+            details={"num_null_rows": 1, "pct_null_rows": 20.0, "rows": [2]},
             action_options=[
                 DataCheckActionOption(
-                    DataCheckActionCode.IMPUTE_COL,
+                    DataCheckActionCode.DROP_ROWS,
                     data_check_name="InvalidTargetDataCheck",
-                    parameters={
```

Comment:
why are we removing all the params?

Reply:
This is testing the default action which is now drop rows, and the drop rows action doesn't have any of the parameters required by the impute column action